### DISCUSSION TITLE
Honeypot backwards compatibality

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#463] [_Discarded pitches]
### Current State
Backwards compatibiliy is not given when the honeypot option is activated by default on existing installations.

If the client has custom CSS the field is visible in frontend. Most of our websites use custom css which means the field will be visible for all our websites if we update the plugin.

### Desired State
The honeypot default value should not be activated for existing installations automatically.

**Comment by dai-eastgate** [comment]
Yes, I will check again Backwards compatibiliy now.

**Reply by dai-eastgate** [reply]
@fredericalpers "The honeypot default value" activated for existing installations automatically. Because I flowed the requirment on this issue https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/446

![image](https://user-images.githubusercontent.com/106214469/224914463-e642a9d6-97f4-41a5-9efe-5e93d08e7415.png)

I understand the Backwards compatibiliy which you give.

If you want "The honeypot default value should not be activated for existing installations automatically."
I will fix it. pls, check again.thanks

**Reply by fredericalpers** [reply]
Thank you for checking. This was a mistake on our end. Please go ahead and fix it, since it's not working properly with custom css. Customers will have to activly decide to enable the honeypot (check the box) and then have to adjust their css if they want to keep using the honeypot.

**Reply by dai-eastgate** [reply]
Ok, I will fix it.

**Reply by dai-eastgate** [reply]
@fredericalpers
We have found solution to resolve "Backwards compatibiliy is not given when the honeypot option is activated by default on existing installations."
Although the honeypot option is activated, it still does not affect If the client has custom CSS the field is visible in frontend.
So we don't need to deactivate it.
please watch the video and give me your opinion on the matter?


https://user-images.githubusercontent.com/106214469/226829853-d107066f-fa0e-4e4b-88c7-f5dc549fe622.mp4

**Reply by fredericalpers** [reply]
@dai-eastgate In my opinion it has to be deactivated still, because we do not know what kind of CSS or maybe even JavaScripts the Themes, Page Builders etc. used by customers and external plugin users are overwriting the plugin CSS. Please go ahead and deactivate it by default, so the customers have to activly enable it.

**Reply by dai-eastgate** [reply]
@fredericalpers I have deactivated the honeypot by default, you can watch the video below and give me your feedback.

https://files.fm/u/q9f7wjswj

**Reply by fredericalpers** [reply]
Looks good to me! As always I'll consult with our Devs and let you know. :)

**Comment by dai-eastgate** [comment]
Merged https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/471
